python/vowpalwabbit/__init__.py

```diff
@@ -1,16 +1,74 @@
 # -*- coding: utf-8 -*-
 """Python interfaces for VW"""
 
-import importlib
+import importlib as _importlib
+import warnings as _warnings
```

Comment:
Why change the name?

Reply:
Since they aren't exports of the package itself I think _ means it is hidden?